src/components/NavBar.tsx

```diff
@@ -0,0 +1,79 @@
+import { navDesktopHeight, navMobileHeight } from '../../src/constants';
+import styled, { css } from 'styled-components';
+import theme from '../../src/theme';
+import OpenstaxLogo from './NavBarLogo';
+
+const BarWrapper = styled.div`
+  overflow: visible;
+  z-index: ${theme.zIndex.navbar};
+  background: ${theme.colors.palette.white};
+  position: relative;
+  padding: 0 ${theme.padding.navbar.mobile}rem;
+  box-shadow: 0 0.2rem 0.2rem 0 rgba(0, 0, 0, 0.1);
+  ${theme.breakpoints.desktop(css`
+    padding: 0 ${theme.padding.navbar.mobile}rem;
+  `)}
+`;
+
+const StyledNavBar = styled.div<{
+  maxWidth?: number;
+  navDesktopHeight: number;
+  navMobileHeight: number;
+}>`
+  overflow: visible;
+  display: flex;
+  justify-content: space-between;
```

Comment:
i feel like space-between is only gonna work if you have a logo and one other child you want on the right side. if you don't have the logo or have more than one child or want it in the center or whatever this isn't going to work. it might be more flexible to remove the justification and allow the child to set `margin-left:auto` if it wants to be on the right.

or maybe make `space-between` the default value for a configurable prop

Reply:
My thought with this was to keep a default case, and export the logo and rely on the consumer making the child container `flex-grow` to assimilate the entire container, which lets you do whatever you want with it. But maybe the style prop to control the parent container is more discoverable, I'll run with that.